custom_components/alexa_media/config_flow.py

```
@@ -687,7 +688,8 @@ async def _test_login(self):
                 )
                 async_dismiss_persistent_notification(
                     self.hass,
-                    notification_id=f"alexa_media_{slugify(email)}{slugify(login.url[7:])}",
+                    host = urlparse(login.url).hostname or login.url
```

Pylint (4.0.5)

[error] Parsing failed: 'invalid syntax. Perhaps you forgot a comma? (alexa_media.config_flow, line 691)'

(E0001)